persid.el

```diff
@@ -95,10 +95,25 @@
 ;; - First implementation
 ;;
 ;;; Code:
+(require 'bibtex)
```

Comment:
Those dependencies should be added as well to the `Package-Requires: ` header line, no?

Reply:
Looks like they are all in Emacs' standard lib already, aren't they? Maybe then just be precise on the first emacs version that allows the package to work (probably 28.2 with json)?